package.json

```diff
@@ -14,17 +14,20 @@
   },
   "dependencies": {
     "@date-io/moment": "1.x",
+    "@emotion/cache": "^11.7.1",
     "@emotion/react": "^11.7.1",
     "@emotion/server": "^11.4.0",
     "@emotion/styled": "^11.6.0",
     "@mui/icons-material": "^5.2.5",
     "@mui/lab": "^5.0.0-alpha.61",
     "@mui/material": "^5.2.5",
     "@stripe/stripe-js": "^1.17.1",
+    "@types/events": "^3.0.0",
     "clsx": "^1.1.1",
+    "events": "^3.3.0",
     "http-status-codes": "^2.1.4",
     "moment": "^2.29.1",
-    "next": "^12.0.7",
+    "next": "12.0.4",
```

Comment:
No idea if downgrading next will change/break anything, @brunohpaiva is this fine? I know you updated it recently so I wanna check to make sure this isn't too far back

Reply:
I don’t think this would break it, it worked when I ran it. But I can switch to a canary version with the fix implemented if that’s ok.

Reply:
I've tested and looks like it works, feels a bit slower to load pages on dev mode but that's my PC I guess